src/logger/loggerTaskEx.c

```diff
@@ -243,8 +247,7 @@ void loggerTaskEx(void *params)
                 }
 
                 /* send the sample on to the telemetry task(s) */
-                if (sampledRate >= telemetrySampleRate ||
-                    currentTicks % telemetrySampleRate == 0)
+                if (!isHigherSampleRate(sampledRate, telemetrySampleRate))
```

Comment:
I'm fine with moving it to 2.8.6, but since we are in this discussion I think we should finish it rather than start it again later.  Help me understand how that mod operator makes much of a difference.  When I was reading the code it seems extraneous.  Do you have an example where this isn't the case?  Possibly if all values are 0, but that is all I can see.